<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="KOWSALYA" ignored="0" name="Test" tests="6" failures="1" timestamp="2022-02-19T13:58:47 IST" time="0.054" errors="0">
  <testcase name="demo" time="0.009" classname="com.testng.Assert_Concept">
    <skipped/>
  </testcase> <!-- demo -->
  <testcase name="demo" time="0.001" classname="com.testng.Assert_Concept">
    <skipped/>
  </testcase> <!-- demo -->
  <testcase name="demo" time="0.001" classname="com.testng.Assert_Concept">
    <skipped/>
  </testcase> <!-- demo -->
  <testcase name="demo" time="0.001" classname="com.testng.Assert_Concept">
    <skipped/>
  </testcase> <!-- demo -->
  <testcase name="demo" time="0.001" classname="com.testng.Assert_Concept">
    <skipped/>
  </testcase> <!-- demo -->
  <testcase name="demo" time="0.001" classname="com.testng.Assert_Concept">
    <failure type="java.lang.AssertionError" message="expected [John] but found [Johnson]">
      <![CDATA[java.lang.AssertionError: expected [John] but found [Johnson]
at com.testng.Assert_Concept.demo(Assert_Concept.java:18)
at java.util.ArrayList.forEach(Unknown Source)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- demo -->
</testsuite> <!-- Test -->
